src/featureflagservice/lib/featureflagservice/feature_flags/feature_flag.ex

```diff
@@ -8,7 +8,7 @@ defmodule Featureflagservice.FeatureFlags.FeatureFlag do
 
   schema "featureflags" do
     field :description, :string
-    field :enabled, :boolean, default: false
+    field :enabled, :float, default: 0.0
```

Comment:
@puckpuck @julianocosta89 doh 🤦 this is the problem here. This needs to stay a boolean until the first migration runs. But in my testing I also realized that using `make start` and `make stop` consistently always removes the volume, so there isn't actually a need to do the migrations idiomatically/split.